### ISSUE TITLE
Freebayes for non-diploid cancer cells

### ISSUE THREAD

**Issue by RegnerM2015** [wheaton5/souporcell#101]
Hi @wheaton5,

If users are working with non-diploid cells, such as cancer cells that are aneuploidy (loss and gain), can users adjust the freebayes step as follows?:

`freebayes -f ref.fa -F 0.01 -C 1 --pooled-continuous aln.bam >var.vcf`

--pooled-continuous should be "agnostic" to the ploidy state and will make frequency-based calls

**Comment by wheaton5** [comment]
Yes this would be appropriate, but it is largely untested and i think very unlikely to work. The noise in the system is very high vs the few somatic mutations that define different cancer cell lines. All of the cancer phylogeny single cell things ive seen have included deep exome sequencing to find the somatic loci and then focus on them. And honestly ive been pretty skeptical of those as well.
